all_views_completions.py

```diff
@@ -51,30 +53,32 @@ def on_query_completions(self, view, prefix, locations):
             contents = w.replace('$', '\\$')
             if v.id != view.id and v.file_name():
                 trigger += '\t(%s)' % basename(v.file_name())
+            if v.id == view.id:
+                trigger += '\tabc'
```

Comment:
@kylebebak what's the meaning of "abc"? The idea of the current implementation is that we show the filename if a completion originates in another file, and none otherwise.

Reply:
Hey @alienhard , thanks for the quick response!

The "abc" I borrowed from VS Code. VS Code adds the "abc" label to auto-complete options for text in the current file.

<img width="812" alt="Screen Shot 2019-06-10 at 12 24 10 AM" src="https://user-images.githubusercontent.com/1524088/59174270-45e9c380-8b16-11e9-9ce3-bae5baf0b97b.png">

I like it because it helps users distinguish between auto-complete options supplied by All Autocomplete vs those supplied by other plugins, even when the suggestions comes from the text in the current view.

If you don't think it's appropriate, or you think some other label text (like "view" or "this" or "here") would be better, just tell me and I'll change it.

IMO the VS Code convention is fairly intuitive.

Reply:
I like this concept. Maybe it could be moved to a setting where people could make their own prefix if needed or disable all together.

Reply:
Let's go with "abc"! I think, even for users who don't know where this is coming from, it's fairly quick and intuitive to realize what it means. I'd rather not add another setting though to keep things simple - but thanks for the suggestion GIB!